src/main/java/com/mateacademy/springmvc/controller/UserController.java

```diff
@@ -0,0 +1,56 @@
+package com.mateacademy.springmvc.controller;
+
+import com.mateacademy.springmvc.entity.User;
+import com.mateacademy.springmvc.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class UserController {
+    private final UserService service;
+
+    @Autowired
+    public UserController(UserService service){
+        this.service=service;
```

Comment:
formate your code

Reply:
you can use constructor injection in Spring with lombok
https://www.baeldung.com/spring-injection-lombok